### PR TITLE
Adds support for version 7 for custom fact and config file testing.

### PR DESCRIPTION
**Pull request by trenta**
#### Pull Request (PR) description
CLI commands have been changed in version 7.

### CODE REVIEW

**Comment by rpf3** [comment]
@trenta, any plans on resurrecting this PR for v7 support?

File: lib/facter/winlogbeat_version.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@
 Facter.add('winlogbeat_version') do
   confine :kernel => %w[Windows] # rubocop:disable Style/HashSyntax
   if File.exist?('c:\Program Files\Winlogbeat\winlogbeat.exe')
-    winlogbeat_version = Facter::Util::Resolution.exec('"c:\Program Files\Winlogbeat\winlogbeat.exe" --version')
+    winlogbeat_version = Facter::Util::Resolution.exec('"c:\Program Files\Winlogbeat\winlogbeat.exe" version')
+    if winlogbeat_version.empty?
+      winlogbeat_version = Facter::Util::Resolution.exec('"c:\Program Files\Winlogbeat\filebeat.exe" --version')
```
Comment (rpf3) [inline lib/facter/winlogbeat_version.rb]:
```suggestion
      winlogbeat_version = Facter::Util::Resolution.exec('"c:\Program Files\Winlogbeat\winlogbeat.exe" --version')
```

**Comment by trenta** [comment]
Hi @rpf3 I've added your fix.  I had to make some other unrelated changes and remove a one syntax check that was an unrelated error to this fix to get the build to work so who knows if it will get merged.

**Comment by bastelfreak** [comment]
@trenta can you please rebase against our master branch?

**Comment by bastelfreak** [comment]
Hi @trenta, there still seems to be a merge commit in your feature branch. Please rebase against our latest master. If you need help with that you can ping us on our IRC channel #voxpupuli on freenode or on slack at https://slack.puppet.com/

**Pull request closed without merging**